guide/src/customization/font-scaling.md

```diff
@@ -0,0 +1,85 @@
+# Font Scaling
+
+By default, Chapbook scales the font size of text based on the width of the
+_viewport_: the container in which the page is viewed. On a desktop computer,
+the viewport is a browser window, which can be resized by the player as much as
+they like. On a tablet or mobile device, the viewport is typically the entire
+screen.
+
+Chapbook will only ever increase the font size you set on text. It will never
+make it smaller. The intention is to keep text at a comfortable size regardless
+of the size of the viewport, and to prevent it from appearing in an unnaturally
+narrow column in larger viewports.
+
+You can customize Chapbook's font scaling or disable it entirely, so that text
+is always exactly at the size you specify.
+
+## Enabling or Disabling Scaling
+
+To disable font scaling, set `config.style.fontScaling.enabled` to `false`. This
+takes effect immediately. To re-enable font scaling, set this variable to
+`true`.
+
+Remember that players also have the ability to scale font sizes by changing
+browser settings. You can't prevent this from occurring--nor should you want to,
+since often players do this to make text easier to read.
+
```

Comment:
Font scaling can also be done at an Operating System lever.
eg. Windows has a setting increasing the base font size of all applications, as well as its desktop.